packages/@storefront/breadcrumbs/CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [patch]
+### Added
+- SF-1248: Created `updateSelectedNavigations` and `getSelectedNavigations` methods in order to generate the new `selectedNavigations` prop.
+- SF-1248: `refinement-crumbs` receives the new `selectedNavigations` prop and uses that to generate its state.
+
+### Changed
+- SF-1248: `refinement-crumbs` now subscribes to the `NAVIGATIONS_UPDATED` event if `selectedNavigations` prop is not available.
+- SF-1248: `refinement-crumbs` now subscribes to the `PAST_PURCHASE_NAVIGATIONS_UPDATED` event if `selectedNavigations` prop is not available.
+
+### Deprecated
+- SF-1248: Marking the following methods and properties from the `breadcrumbs` component for deprecation:
+  -  Methods to be deprecated: 
```

Comment:
Don't speak as if this is going to be done. To the reader, this is already done.

Reply:
Understood, will make the changes. Resolving conversation.